plugins.d/python.d.plugin

```diff
@@ -255,6 +255,7 @@ class PythonCharts(object):
         # get default values
         defaults = {}
         for key in BASE_CONFIG:
+            # FIXME for some reason this is called 3 times per module
             debug(module.__name__ + ": reading configuration")
```

Comment:
`debug()` should be before `for`. This loop looks and sets up default module configuration (`update_every`, `retries`, `priority`)